### DISCUSSION TITLE
Should we release 0.73.2?

### DISCUSSION THREAD

**Discussion by huntie** [reactwg/react-native-releases#95] [Patches]
## Released, please refer to #98

Current Release: [0.73.1](https://github.com/facebook/react-native/releases/tag/v0.73.1)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

**Comment by huntie** [comment]
To cherry pick (carried over from https://github.com/reactwg/react-native-releases/discussions/94):

- https://github.com/facebook/react-native/commit/1a0e1741c0ab4f22b57a6a7ee86ff4287fa09103

**Comment by szymonrybczak** [comment]
https://github.com/facebook/react-native/pull/41930 - small change in template, allowing CLI to properly replace project name.

**Reply by cortinico** [reply]
This is not needed anymore as it has been closed 👍

**Comment by cipolleschi** [comment]
CircleCI got broken yesterday due to Ruby and bundler. This is the fix:
https://github.com/facebook/react-native/commit/479b4bb1ba504706613f71530a4ddcf86ff1bcd9

**Reply by lunaleaps** [reply]
✅ merged

**Comment by axinvd** [comment]
Android border platformcolor fix
Already included in 0.72.7, but not present in 73.x
https://github.com/facebook/react-native/commit/265af222aa8acb3b514e22a7db191d66755f553b

**Reply by huntie** [reply]
Picked ✅

**Comment by lunaleaps** [comment]
We need to delete the relevant translation folders that breaks deploying to Play store. 
https://github.com/facebook/react-native/issues/42024
@NickGerleman has fixed main to exclude this from being included on `main`

**Reply by lunaleaps** [reply]
https://github.com/facebook/react-native/pull/42040, pending to see if @NickGerleman does more clean-up via an allow-list vs. just these two problematic translations

**Reply by lunaleaps** [reply]
✅ I've just merged this for now, we can delete more later as identified

**Comment by cortinico** [comment]
There are a couple of minor fixes that are needed to be picked in 0.73.2:
- https://github.com/facebook/react-native/pull/41856
- https://github.com/facebook/react-native/pull/41839

**Reply by huntie** [reply]
Picked ✅

**Comment by cipolleschi** [comment]
Can we pick: https://github.com/facebook/react-native/commit/1e5e3b518e868a0b28b3513b084c0dda31c85449?

**Reply by fortmarek** [reply]
Picked ✅

**Comment by pipo151086** [comment]
hi....

i dont know if this is the right place but there has been a issue for a while and there has no been any progress....

https://github.com/crazycodeboy/react-native-splash-screen/issues/634

**Reply by cortinico** [reply]
Nope this is not the right place. Please reach out to the library maintainer directly

**Comment by lunaleaps** [comment]
Two things that are slightly related:
1. https://github.com/facebook/react-native/pull/41958 -- Remove the rest of the `^` in our monorepo deps. This is causing problems because all monorepo dep versions are <1.0 which means `~` and `^` behave the same way and when we have something like `^0.72.2` in our package.json, and we publish`0.73.0`, people will pull that one in erroneously.
2. https://github.com/facebook/react-native/pull/42082 -- We want to remove `*` dependencies for our monorepo deps as this will cause similar issues to 1. We fix these to the latest published versions. 

Both of these changes have also been made on `main`.

**Reply by lunaleaps** [reply]
✅ I've merged both of these

**Comment by cipolleschi** [comment]
Can we merge: https://github.com/facebook/react-native/pull/42087 
This should fix iOS CI for e2e tests

**Reply by lunaleaps** [reply]
✅ merged

**Comment by mmmoussa** [comment]
[In the release blog for 0.73 it was stated that 0.73 would be the last RN version to support Android SDK 21](https://reactnative.dev/blog/2023/12/06/0.73-debugging-improvements-stable-symlinks#min-sdk-bump). However, 0.73 was released with [a change that breaks usage below Android SDK 23](https://github.com/facebook/react-native/commit/1904e8036a777e463c5517010a254f0d081ae58c). That commit should be reverted in 0.73.2.

**Reply by NickGerleman** [reply]
Whoah, IDK how that change got picked into the 0.73 branch. https://github.com/facebook/react-native/commit/40fe1cbd8c61d921b1fb19cbbc7fcb6dc0f8a181

I didn't see it in the comments for 0.73.0 picks at least. @lunaleaps remember the context on this change?

**Reply by cortinico** [reply]
That's definitely a problem and should be reverted/addressed. cc @mdvacca aslo

**Reply by huntie** [reply]
Reverted ✅

**Reply by cortinico** [reply]
@huntie small heads up that I don't think we can simply revert this. There should probably be a fix forward which I'd like to discuss with @mdvacca

**Reply by mdvacca** [reply]
After analyzing this with @cortinico, we concluded that:
- https://github.com/facebook/react-native/commit/1904e8036a777e463c5517010a254f0d081ae58c was incorrectly picked into RC, so is ok to just revert it.
- we have to make sure https://github.com/facebook/react-native/commit/45dce9853348fdf4c1f03c0edcddf8c7ce94d6a0 is actually in the 0.73 (this was the correct diff to pick into 0.73)

cc: @huntie

**Reply by huntie** [reply]
@mdvacca @cortinico Thanks, actioned ✅

**Comment by jacobp100** [comment]
Can we get https://github.com/facebook/react-native/commit/d6163d7f43a753f24148450033491dfc9c5d0c36 ?

**Reply by huntie** [reply]
Picked ✅

**Comment by joe-sam** [comment]
Corrects LogBox to display the actual stack trace libraries instead of internal code on the debug thread for all custom errors during unhandled promises.
https://github.com/facebook/react-native/pull/42079

**Reply by huntie** [reply]
Picked ✅

**Comment by sagarj-iprogrammer** [comment]
can we pick 
https://github.com/facebook/react-native/issues/36710
@huntie  same issue with RN0.73.1

**Comment by NickGerleman** [comment]
Could we pick https://github.com/facebook/react-native/commit/e809e0aca713ad12f91778b09f5297f1da866b26 to fix RTL scrollTo issue on iOS?

**Reply by huntie** [reply]
Picked ✅

**Comment by vikrambatra08** [comment]
Can we pick - https://github.com/facebook/react-native/issues/34400 ?

**Reply by liamjones** [reply]
I believe Vikram means this commit specifically: https://github.com/facebook/react-native/commit/bab9c839eaad1ffc358fa5c87440a6de7cbe981b

**Reply by huntie** [reply]
@cortinico @cipolleschi Are we happy to bump SocketRocket on both platforms?

**Reply by liamjones** [reply]
@huntie Sorry, I should've looked at the PR before suggesting Vikram add a pick request here. Looking back at the original PR it looks like @cipolleschi already said it'd have to wait until Flipper is gone (0.74) because of issues with the update + Flipper on Android: https://github.com/facebook/react-native/pull/39571#issuecomment-1731125611 & https://github.com/facebook/react-native/pull/39571#issuecomment-1731765114

**Reply by kelset** [reply]
also I don't think we'd want to bump between majors (0.6.x -> 0.7.x) on a patch release (which is why it even the commit body mentions that it would have been only viable to pick up in 0.73 before 0.73.0)
